src/plugins/unified_search/public/query_string_input/query_string_input.tsx

```diff
@@ -725,7 +725,7 @@ export default class QueryStringInputUI extends PureComponent<Props, State> {
                 placeholder={
                   this.props.placeholder ||
                   i18n.translate('unifiedSearch.query.queryBar.searchInputPlaceholder', {
-                    defaultMessage: 'Search',
+                    defaultMessage: 'Start typing to search or filter...',
```

Comment:
Is it important to say both search and filter here?  Will the user understand the difference?

Also, type might not be the right term here, since the user can also select terms and copy and paste.

Also for the menu, is it clear to the user what "Apply all" and "Clear all" apply to?

Reply:
@cchaos @ghudgins what do you think about it? I think that the user testing is going to also help on these.

Reply:
I'd like to find text that provokes understanding that this is specifically for filtering down the data that exists on the page. That it is separate from the global search which is used for navigation. Simply `Search` is just not enough. Even more than this, I think we need to indicate the current filtering language that is selected in this placeholder text. We used to have the language switcher visible at all times, but since we've moved it to a less visible location, we should indicate that the search input requires certain formatting.

Reply:
Thanks for the description, @cchaos.  Here is a suggestion for the placeholder text:

Filter your data using xxx syntax

where xxx is KQL, Lucene, etc.

Reply:
I like that for specific language syntaxes, but there's also an option for the search bar to be plain text. I don't think we would want

> Filter your data using text syntax

Maybe for plain text it's just..?

> Filter you data

Reply:
That works.

Would the `Filter your data` text ever be close to the `Add filter` button, as in Discover?

Reply:
This is where the `Add filter` button now lives:
![Screen Shot 2022-04-06 at 18 04 21 PM](https://user-images.githubusercontent.com/549577/162079930-acbfb1bf-ea9f-48cd-bd1c-f6b716691c96.png)